openmls/src/messages/mod.rs

```diff
@@ -262,19 +258,19 @@ impl GroupInfo {
     }
 
     /// Returns other application extensions.
-    pub(crate) fn other_extensions(&self) -> &[Extension] {
-        self.payload.other_extensions.as_slice()
+    pub(crate) fn extensions(&self) -> &[Extension] {
```

Comment:
This does not return the extensions of the group context but the extensions of the `GroupInfoTBS`. That's why it was called `other_extensions`.
If you want to keep this name, you should remove functions returning values from the `group_context`, return only a reference to the `group_context`, and let the caller get the value. And you should rename `set_other_extensions` in this case.

Reply:
I cleaned this up.